### PR TITLE
Updates for Ocean 4

### PR DESCRIPTION
**Pull request by hhtong**

### CODE REVIEW

File: tests/test_jn.py [hunk 1]
```diff
@@ -81,7 +81,8 @@ def test_jn(self):
         self.assertIn('(0, 0, 0, 0)', nb["cells"][9]["outputs"][1]["data"]['text/plain'])
 
         # Section Step 2: Convert to a BQM, code cell 2, print(and_bqm.quadratic)
-        self.assertIn('(\'x2\', \'x3\')', cell_text(nb, 11))
+        cell_output = cell_text(nb, 11)
+        self.assertTrue('(\'x3\', \'x2\')' in cell_output or '(\'x2\', \'x3\')' in cell_output)
```
Comment (randomir) [inline tests/test_jn.py]:
```suggestion
        self.assertTrue("('x3', 'x2')" in cell_output or "('x2', 'x3')" in cell_output)
```

File: 01-factoring-overview.ipynb [hunk 2]
```diff
@@ -188,7 +188,7 @@
    "outputs": [],
    "source": [
     "and_bqm = dbc.stitch(and_csp)\n",
-    "and_bqm.remove_offset()\n",
+    "and_bqm.offset = 0\n",
```
Comment (JoelPasvolsky) [inline 01-factoring-overview.ipynb:191]: What's the reason for this change?

**Review by randomir — APPROVED** [review]

File: 01-factoring-overview.ipynb (hunk 2 above)
Comment (randomir) [reply 01-factoring-overview.ipynb:191]: `.remove_offset` is removed in dimod 0.10

Comment (JoelPasvolsky) [reply 01-factoring-overview.ipynb:191]: I guessed that might be the case but when I look at https://github.com/dwavesystems/dimod/blob/b8a91bd810d632ace95efa90fba7993d918a8a45/dimod/core/bqm.py#L462 I didn't see a deprecation. Is that somewhere else?

Comment (randomir) [reply 01-factoring-overview.ipynb:191]: @arcondello might know.

Comment (arcondello) [reply 01-factoring-overview.ipynb:191]: No, 0.9 -> 0.10 is a backwards breaking change afterall and this is one of those changes. If it's a big problem we can restore it with a deprecation warning but we'll still need to change it here to avoid the warning.

Comment (JoelPasvolsky) [reply 01-factoring-overview.ipynb:191]: I suspect this function might have been used similarly in other code. Should we maybe have a deprecation warning now and take it out in the next release?

Comment (arcondello) [reply 01-factoring-overview.ipynb:191]: Sure, though we still need to make this change here and everywhere else to avoid raising the warning.

**Review by JoelPasvolsky — APPROVED** [review]
